### PR TITLE
Tinker with constants for drawing the define block.

### PR DESCRIPTION
**Pull request by rachel-fenichel**
### Resolves

Last piece of https://github.com/LLK/scratch-blocks/issues/1102

### Proposed Changes

Get the size of the hat right on the define block.

### Reason for Changes

Matching the spec.

### Test Coverage

image
image

### CODE REVIEW

**Comment by rachel-fenichel** [comment]
cc @carljbowman I *think* this is right now.

File: core/block_render_svg_vertical.js [hunk 1]
```diff
@@ -1146,7 +1156,7 @@ Blockly.BlockSvg.prototype.renderDrawTop_ = function(steps, rightEdge) {
   /* eslint-disable indent */
   if (this.type == Blockly.BlockSvg.DEFINE_BLOCK_TYPE) {
     steps.push('m 0, 0');
-    steps.push('v', -2 * Blockly.BlockSvg.GRID_UNIT);
+    //steps.push('v', -2 * Blockly.BlockSvg.GRID_UNIT);
```
Comment (paulkaplan) [inline core/block_render_svg_vertical.js]: Should this line be removed instead of commented out?

**Review by paulkaplan — APPROVED** [review]
Code LG (other than that one commented line).

File: core/block_render_svg_vertical.js (hunk 1 above)
Comment (rachel-fenichel) [reply core/block_render_svg_vertical.js]: Good catch.  Done.

**Pull request merged**